holdings_maintenance/pds4/pds4checksums.py

```
@@ -0,0 +1,895 @@
+                del md5_dict[abspath]
+                logger.normal('Validated', abspath)
+
+        if not selection:
+            abspaths = list(md5_dict.keys())
+            abspaths.sort()
+            for abspath in abspaths:
+                logger.error('Extra file', abspath)
+                success = False
+
+    except (Exception, KeyboardInterrupt) as e:
+        logger.exception(e)
```

Ruff (0.14.5)

Redundant exception object included in `logging.exception` call

(TRY401)